### DISCUSSION TITLE
Support IntelliJ 2023.3 EAP

### DISCUSSION THREAD

**Discussion by glains** [community/community#68032] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Currently, Copilot build `1.2.22.3249-nightly` does not support IntelliJ 2023.3 EAP, which offers better support for Java 21.
When is support to be expected?

**Comment by Akash1134** [comment]
Hi @glains ,

Thank you for posting in GitHub Community.

We support only the latest stable version of the JetBrains IDEs. However, our team is aware of the new early access build -- IntelliJ 2023.3 EAP. 

We will update this discussion as soon as we support it officially. 

Thanks!

**Comment by glains** [comment, marked as answer]
Thank for the work. To my knowledge, Copilot version `1.3.0.3333-nightly` works with IntelliJ 2023.3 EAP.
